server/src/test/java/org/opensearch/cluster/node/DiscoveryNodesTests.java

```diff
@@ -108,12 +108,12 @@ public void testAll() {
         assertThat(discoveryNodes.resolveNodes(new String[0]), arrayContainingInAnyOrder(allNodes));
         assertThat(discoveryNodes.resolveNodes("_all"), arrayContainingInAnyOrder(allNodes));
 
-        final String[] nonMasterNodes = StreamSupport.stream(discoveryNodes.getNodes().values().spliterator(), false)
+        final String[] nonClusterManagerNodes = StreamSupport.stream(discoveryNodes.getNodes().values().spliterator(), false)
             .map(n -> n.value)
             .filter(n -> n.isMasterNode() == false)
             .map(DiscoveryNode::getId)
             .toArray(String[]::new);
-        assertThat(discoveryNodes.resolveNodes("_all", "master:false"), arrayContainingInAnyOrder(nonMasterNodes));
+        assertThat(discoveryNodes.resolveNodes("_all", "master:false"), arrayContainingInAnyOrder(nonClusterManagerNodes));
```

Comment:
`master` -> `clusterManager`

Reply:
Thanks! I added a new test for validating the deprecated "master:false" node filter, and renamed "master:false" here to "cluster_manager:false" which is corresponding with the definition "[DiscoveryNodes.resolveNodes()](https://github.com/opensearch-project/OpenSearch/blob/2.0.0-rc1/server/src/main/java/org/opensearch/cluster/node/DiscoveryNodes.java#L381)"